cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/replication/AttachmentsPullTest.java

```diff
@@ -109,7 +109,7 @@ public void pullRevisionsWithBigAttachments() throws Exception {
         Assert.assertNotNull("Attachment is null", a);
         Assert.assertEquals(bigAttachmentName, a.name);
         try {
-            Assert.assertTrue("Streams not equal", TestUtils.streamsEqual(new FileInputStream(TestUtils.loadFixture("fixture/"+bigAttachmentName)), a.getInputStream()));
+            Assert.assertTrue("Streams not equal", TestUtils.streamsEqual(new FileInputStream(TestUtils.loadFixture("fixture/" + bigAttachmentName)), a.getInputStream()));
```

Comment:
Any chance that we can break the assertTrue statement to multiple lines?  When viewing the file in GH I had to scroll to the right to see the full statement.